net/probe-cli/patches-probe-engine/003-fix-sum.patch

```diff
@@ -0,0 +1,20 @@
+--- a/go.sum
```

Comment:
A small nitpick: Would you mind combining the changes to go.mod and go.sum into one patch? They are related/connected changes.

Reply:
Fixed